README.md

```diff
@@ -161,20 +161,25 @@ https://console.developers.google.com: click on "Select a project..."
 in the top navbar and then "Create a project."
 
 You will then need a name for your AppEngine project.  Let's suppose
-you call it `mycompany-snippets`.
+you call it `mycompany-snippets`. You must then set the project by 
+running `gcloud config set project mycompany-snippets` from the snippets
```

Comment:
Hrm, that totally got me -- the output from the tooling when attempting to run against an old-style config totally makes it seem like it's a directory-specific thing, the way it works with almost every other cloud platform tool (heroku, DO, etc). That's a strange decision.

Will fix.